tools/export.py

```diff
@@ -96,8 +97,13 @@ def check_args(args):
         type=str,
         default="",
         help='Directory to save the exported mindir file.')
+    parser.add_argument(
+        '--amp_level',
```

Comment:
If user input a yaml file path (the amp_level is set in yaml) to export mindir, he supposes all the args in yaml is loaded when exporting. However, we set default amp_level to be O0 here, which may be different in the one in user's yaml. This may cause misleading to user.
It's suggested to set --amp_level `required=True`. Add `choices=['O0', 'O1', 'O2', 'O3']`would be better. :)